photos_gallery/gallery/templates/gallery/base.html

```diff
@@ -40,7 +40,7 @@
             <img class="logo_" src="https://www.w3schools.com/images/w3schools_green.jpg" title="IMAGE_TEST_NAME">
         </a>
         <a class="own" href="#">Sign In / </a>
-        <a class="own" href="">Register</a>
+        <a class="own" href="/register">Register</a>
```

Comment:
Use django url helper
```suggestion
        <a class="own" href="{% url 'register' %}">Register</a>
```

Reply:
See https://docs.djangoproject.com/en/4.0/ref/templates/builtins/#url and https://docs.djangoproject.com/en/4.0/topics/http/urls/#reversing-namespaced-urls for the explanation how to use url.py and the url helper in templates.